src/geo/LatLng.js

```diff
@@ -122,16 +133,17 @@ export function toLatLng(a, b, c) {
 		if (a.length === 2) {
 			return new LatLng(a[0], a[1]);
 		}
-		return null;
-	}
-	if (a === undefined || a === null) {
-		return a;
 	}
 	if (typeof a === 'object' && 'lat' in a) {
-		return new LatLng(a.lat, 'lng' in a ? a.lng : a.lon, a.alt);
+		var lat = a.lat,
+		    lng = 'lng' in a ? a.lng : a.lon;
+		if ('alt' in a) {
+			return new LatLng(lat, lng, a.alt);
+		}
+		return new LatLng(a.lat, 'lng' in a ? a.lng : a.lon);
 	}
-	if (b === undefined) {
-		return null;
+	if (arguments.length === 3) {

```

Comment:
With the check `arguments.length == 3` it is a breaking change which can affect many projects. I think it is a valid change but we need to keep this in mind